golem-worker-executor-base/src/services/rdbms/tests.rs

```diff
@@ -0,0 +1,2938 @@
+                timestamp_col TIMESTAMP,
+                timestamptz_col TIMESTAMP WITH TIME ZONE,
+                date_col DATE,
+                time_col TIME,
+                timetz_col TIME WITH TIME ZONE,
+                interval_col INTERVAL,
+                bytea_col BYTEA,
+                uuid_col UUID,
+                json_col JSON,
+                jsonb_col JSONB,
+                inet_col INET,
+                cidr_col CIDR,
+                macaddr_col MACADDR,
+                bit_col BIT(3),
+                varbit_col VARBIT,
+                xml_col XML,
+                int4range_col INT4RANGE,
+                int8range_col INT8RANGE,
+                numrange_col NUMRANGE,
+                tsrange_col TSRANGE,
+                tstzrange_col TSTZRANGE,
+                daterange_col DATERANGE,
+                money_col MONEY,
+                jsonpath_col JSONPATH,
+                tsvector_col TSVECTOR,
+                tsquery_col TSQUERY,
+                inventory_item_col inventory_item,
+                posint4_col posint4,
+                float8range_col float8range
+            );
+        "#,
+    ];
+
+    rdbms_test(
+        rdbms.clone(),
+        &db_address,
+        RdbmsTest::new(
+            statements
+                .into_iter()
+                .map(|s| StatementTest::execute_test(s, vec![], Some(0)))
+                .collect(),
+            None,
+        ),
+    )
+    .await;
+
+    let insert_statement = r#"
+            INSERT INTO data_types
+            (
+            id,
+            enum_col,
+            char_col,
+            int2_col,
+            int4_col,
+            int8_col,
+            float4_col,
+            float8_col,
+            numeric_col,
+            boolean_col,
+            text_col,
+            varchar_col,
+            bpchar_col,
+            timestamp_col,
+            timestamptz_col,
+            date_col,
+            time_col,
+            timetz_col,
+            interval_col,
+            bytea_col,
+            uuid_col,
+            json_col,
+            jsonb_col,
+            inet_col,
+            cidr_col,
+            macaddr_col,
+            bit_col,
+            varbit_col,
+            xml_col,
+            int4range_col,
+            int8range_col,
+            numrange_col,
+            tsrange_col,
+            tstzrange_col,
+            daterange_col,
+            money_col,
+            jsonpath_col,
+            tsvector_col,
+            tsquery_col,
+            inventory_item_col,
+            posint4_col,
+            float8range_col
+            )
+            VALUES
+            (
+                $1, $2, $3, $4, $5, $6, $7, $8, $9,
+                $10, $11, $12, $13, $14, $15, $16, $17, $18, $19,
+                $20, $21, $22, $23, $24, $25, $26, $27, $28, $29,
+                $30, $31, $32, $33, $34, $35, $36, $37, $38::tsvector, $39::tsquery,
+                $40, $41, $42
+            );
+        "#;
```

Comment:
Was these required because of the type conversions that we have?

Reply:
yes, to test conversions, and type mappings